packages/swarm-components/src/NumericalInput.jsx

```diff
@@ -19,11 +19,11 @@ type Props = {
 	/**
 	 * the maximum integer allowed
 	 */
-	max?: number,
+	max?: ?number,
```

Comment:
Why are these Maybe types? Aren't `min` and `max` always numbers?

Reply:
They can be null or undefined since they are optional props.

Reply:
They're necessary since we try to access the property ex: `this.props.min` -> https://github.com/meetup/swarm-ui/pull/104/files/404fe41624e24c90f4a12738c107b0cf83ab734b#diff-8a350deaf4d5bbb900b072c47b2b99a5R82